sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -901,26 +901,84 @@ __SYCL_EXPORT bool is_source_kernel_bundle_supported(backend BE,
                                                      source_language Language);
 
 __SYCL_EXPORT kernel_bundle<bundle_state::ext_oneapi_source>
+make_kernel_bundle_from_source(
+    const context &SyclContext, source_language Language,
+    sycl::detail::string_view Source,
+    std::vector<std::pair<sycl::detail::string_view, sycl::detail::string_view>>
+        IncludePairsVec);
```

Comment:
The free functions that are not part of the SYCL specification should be in the `detail` namespace to make it clear that it is not to be used by the user directly.

Reply:
These are already in the namespace 'detail'.
Please see a few lines above (L898)